.textlintrc.js

```diff
@@ -11,8 +11,9 @@ module.exports = {
     "no-empty-section": {
     },
     "rousseau": {
-      "ignoreTypes": ["sentence:uppercase"],
-      "showLevels": ["suggestion", "warning", "error"]
+      "ignoreTypes": [],
```

Comment:
Don't we want to keep `sentence:uppercase` in `ignoreTypes` for sentences that begin with technical terms that may be lowercase?